hie.yaml

```diff
@@ -1,74 +1,95 @@
 cradle:
   stack:
-    - path: "codebase2/codebase"
+    - path: "codebase2/codebase/./"
       component: "unison-codebase:lib"
 
-    - path: "codebase2/codebase-sqlite"
+    - path: "codebase2/codebase-sqlite/./"
       component: "unison-codebase-sqlite:lib"
 
-    - path: "codebase2/codebase-sync"
+    - path: "codebase2/codebase-sync/./"
       component: "unison-codebase-sync:lib"
 
-    - path: "codebase2/core"
+    - path: "codebase2/core/./"
       component: "unison-core:lib"
 
-    - path: "codebase2/util/bench"
-      component: "unison-util:bench:bench"
-
     - path: "codebase2/util/src"
       component: "unison-util:lib"
 
-    - path: "codebase2/util-serialization"
+    - path: "codebase2/util/bench/Main.hs"
+      component: "unison-util:bench:bench"
+
+    - path: "codebase2/util-serialization/./"
       component: "unison-util-serialization:lib"
 
-    - path: "codebase2/util-term"
+    - path: "codebase2/util-term/./"
       component: "unison-util-term:lib"
 
     - path: "lib/unison-prelude/src"
       component: "unison-prelude:lib"
 
+    - path: "lib/unison-pretty-printer/src"
+      component: "unison-pretty-printer:lib"
+
+    - path: "lib/unison-pretty-printer/prettyprintdemo/Main.hs"
+      component: "unison-pretty-printer:exe:prettyprintdemo"
+
+    - path: "lib/unison-pretty-printer/tests"
+      component: "unison-pretty-printer:test:pretty-printer-tests"
+
     - path: "lib/unison-sqlite/src"
       component: "unison-sqlite:lib"
 
-    - path: "unison-share-api/src"
-      component: "unison-share-api:lib"
-
     - path: "lib/unison-util-relation/src"
       component: "unison-util-relation:lib"
 
     - path: "lib/unison-util-relation/test"
-      component: "unison-util-relation:test:tests"
+      component: "unison-util-relation:test:util-relation-tests"
```

Comment:
I don't see where you re-name the appropriate test-suite in this PR; or was this just incorrectly named in the first place?

Reply:
The latter, it seems